<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Shadowsocks-Qt5</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icon/shadowsocks-qt5.png</normaloff>:/icon/shadowsocks-qt5.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="generalTab">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2" rowstretch="10,1,0">
        <item row="2" column="2" alignment="Qt::AlignBottom">
         <widget class="QDialogButtonBox" name="profileEditButtonBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2">
         <layout class="QGridLayout" name="gridLayout_r">
          <item row="4" column="1">
           <widget class="QComboBox" name="methodComboBox">
            <item>
             <property name="text">
              <string notr="true" extracomment="Default">Table</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">RC4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">AES-128-CFB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">AES-192-CFB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">AES-256-CFB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">BF-CFB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">CAMELLIA-128-CFB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">CAMELLIA-192-CFB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">CAMELLIA-256-CFB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">CAST5-CFB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">DES-CFB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">IDEA-CFB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">RC2-CFB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">SEED-CFB</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="methodLabel">
            <property name="text">
             <string>Encryption Method</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="sportLabel">
            <property name="text">
             <string>Server Port</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="pwdEdit">
            <property name="echoMode">
             <enum>QLineEdit::PasswordEchoOnEdit</enum>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="pwdLabel">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="timeoutLabel">
            <property name="text">
             <string>Timeout</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="sportEdit"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lportEdit"/>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="timeoutSpinBox">
            <property name="correctionMode">
             <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
            </property>
            <property name="suffix">
             <string notr="true"> second(s)</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>3600</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lportLabel">
            <property name="text">
             <string>Local Port</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="serverLabel">
            <property name="text">
             <string>Server Domain/IP</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="serverEdit"/>
          </item>
         </layout>
        </item>
        <item row="0" column="1" rowspan="3">
         <widget class="Line" name="vLine">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="startButton">
            <property name="text">
             <string>Start</string>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopButton">
            <property name="text">
             <string>Stop</string>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0" rowspan="2">
         <layout class="QGridLayout" name="gridLayout_l">
          <item row="0" column="3">
           <widget class="QToolButton" name="addProfileButton">
            <property name="text">
             <string>+</string>
            </property>
            <property name="icon">
             <iconset theme="list-add">
              <normaloff/>
             </iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="profileLabel">
            <property name="text">
             <string>Profile</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QComboBox" name="profileComboBox"/>
          </item>
          <item row="0" column="4">
           <widget class="QToolButton" name="delProfileButton">
            <property name="text">
             <string>-</string>
            </property>
            <property name="icon">
             <iconset theme="list-remove">
              <normaloff/>
             </iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="3">
           <widget class="QLineEdit" name="backendEdit">
            <property name="placeholderText">
             <string>Full path to ss-local</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="backendLabel">
            <property name="text">
             <string>Backend</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="backendTypeLabel">
            <property name="text">
             <string>Backend Type</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="4">
           <widget class="QComboBox" name="backendTypeCombo">
            <item>
             <property name="text">
              <string>Shadowsocks-libev</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Shadowsocks-Go</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QToolButton" name="backendToolButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="document-open">
              <normaloff/>
             </iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="logTab">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextBrowser" name="logBrowser"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="miscTab">
       <attribute name="title">
        <string>Misc</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="1">
         <widget class="QRadioButton" name="roundedRadio">
          <property name="text">
           <string>Rounded</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QRadioButton" name="triangularRadio">
          <property name="text">
           <string>Triangular</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="tabStyleLabel">
          <property name="text">
           <string>Tab Style</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QCheckBox" name="autostartCheck">
          <property name="text">
           <string>Autostart if profile is valid.</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="3" alignment="Qt::AlignBottom">
         <widget class="QDialogButtonBox" name="miscButtonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QCheckBox" name="debugCheck">
          <property name="text">
           <string>Debug-level log</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QCheckBox" name="autohideCheck">
          <property name="text">
           <string>Auto hide main window when started.</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
